homeassistant/components/discord/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+    "config": {
+      "step": {
+        "user": {
+          "title": "Discord Notifications",
+          "description": "Refer to the documentation on getting your Discord bot key.\n\nhttps://www.home-assistant.io/integrations/discord",
```

Comment:
Make the URL a placeholder and set it in the flow.
We don't need to translate the actual URL and it makes it easier to change it (if needed in the future).